### ISSUE TITLE
v2.29.0-beta.1 Reloading Metadata problem with column that is an array of text enums

### ISSUE THREAD

**Issue by LydiaF** [hasura/graphql-engine#9748]
Hi, I am trying to out 2.29.0-beta.1 for the postgres arrays feature but running into difficulty when used with array of enums. While the feature worked great with array of text, I cannot reload metadata at all now.

### Version Information

v2.28.0 (not failing) and v2.29.0-beta.1 (failing)

### Environment

Docker

### What is the current behaviour?

Md is failing to reload after creating a new enum array column on 2.29.0-beta.1.

### What is the expected behaviour?

Metadata can reload as it does in 2.28.0.

```
FATA[0008] failed to reload metadata: cannot reload metadata: {
  "error": "cannot use SQL type \"my_col_enum[]\" in the GraphQL schema because its name is not a valid GraphQL identifier",
  "path": "$.args",
  "code": "validation-failed"
} 
```

### How to reproduce the issue?

1. Create a new column of text enums
2. Try to reload the metadata in 2.29.0-beta.1

### Any possible solutions/workarounds you're aware of?

No

### Keywords

metadata
enum array

Thanks :)

**Comment by danieljharvey** [comment]
Hi! Thanks for submitting this and including a clear reproduction. This should be fixed in the upcoming 2.29 release.

**Comment by LydiaF** [comment]
Brill, thanks.
